pkg/util/fileutil/file.go

```diff
@@ -17,13 +17,30 @@
 package fileutil
 
 import (
+	"fmt"
 	"os"
+	"path"
 	"path/filepath"
+	"strings"
 
 	"github.com/cockroachdb/errors"
 	"github.com/sirupsen/logrus"
 )
 
+var (
+	DefaultConfigDir string
+	DefaultCacheDir  string
+)
+
+func init() {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		panic(err)
+	}
+	DefaultConfigDir = path.Join(home, ".config/envd")
+	DefaultCacheDir = path.Join(home, ".config/envd")
```

Comment:
It is still `.config`. I think it should be `.cache`

Reply:
Sorry. Rebased the wrong commit.